crates/walrus-sdk/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Walrus Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! The Walrus Rust SDK.
+
+pub mod active_committees;
+pub mod blocklist;
+pub mod client;
+pub mod config;
+pub mod error;
+pub mod store_when;
+/// Utilities for the Walrus SDK.
+pub mod utils;
+
+pub use sui_types::event::EventID;
+pub use walrus_sui as sui;
+
+/// Format the event ID as the transaction digest and the sequence number.
+pub fn format_event_id(event_id: &EventID) -> String {
```

Comment:
nit: this inside lib.rs feels weird. Can this go to walrus-utils? If so, just add a TODO here and address it later.

Reply:
walrus-utils does not import EventID (sui-types). Having this here is a bit strange, I agree. I'll leave here for now, and we can easily move it later if this placement becomes problematic.